tasklist/client/src/modules/queries/useLicense.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useQuery} from '@tanstack/react-query';
+import {api} from 'modules/api';
+import {RequestError, request} from 'modules/request';
+import {License} from 'modules/types';
+
+function useLicense() {
```

Comment:
[We can add the same query options as in `useCurrentUser` since this data will never change per deployment](https://github.com/camunda/camunda/blob/main/tasklist/client/src/modules/queries/useCurrentUser.tsx)
```
   gcTime: Infinity,
    staleTime: Infinity,
    refetchIntervalInBackground: false,
    refetchOnWindowFocus: false,
```

Reply:
I added them!